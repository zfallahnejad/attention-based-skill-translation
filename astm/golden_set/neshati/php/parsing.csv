118068,parsing
367456,parsing
208809,parsing
345031,parsing
187606,parsing
476,parsing
138475,parsing
1491895,parsing
1226894,parsing
2265374,parsing
53114,parsing
1338292,parsing
324584,parsing
966023,parsing
1438393,parsing
215966,parsing
31671,parsing
283366,parsing
358679,parsing
204701,parsing
4833,parsing
