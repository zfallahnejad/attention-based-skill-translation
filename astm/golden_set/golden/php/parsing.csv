367456,21,29,0.42
118068,21,25,0.45652173913043476
208809,18,18,0.5
345031,15,17,0.46875
187606,12,12,0.5
476,12,7,0.631578947368421
1491895,11,1,0.9166666666666666
138475,11,11,0.5
1226894,10,7,0.5882352941176471
2265374,9,11,0.45
53114,8,7,0.5333333333333333
966023,8,7,0.5333333333333333
324584,8,8,0.5
1338292,8,5,0.6153846153846154
204701,7,2,0.7777777777777778
358679,7,7,0.5
4833,7,10,0.4117647058823529
31671,7,6,0.5384615384615384
283366,7,8,0.4666666666666667
215966,7,5,0.5833333333333334
1438393,7,7,0.5
